pgoutput/Dockerfile

```diff
@@ -4,7 +4,8 @@ RUN mkdir /src
 COPY . /src/pgoutput
 # Contents of "scripts" fix pg_hba.conf and postgresql.conf
 COPY scripts /docker-entrypoint-initdb.d
-
+RUN chmod +r /docker-entrypoint-initdb.d/*
```

Comment:
That works for me. "a+r" makes it so everyone can read. Works great.

So LGTM and I'm going to merge this.

It'd be fun to unravel the mystery of why this breaks, but for another time...